master/buildbot/config.py

```diff
@@ -816,6 +816,7 @@ def __init__(self, name=None, slavename=None, startSlavenames=None, slavenames=N
         self.mergeRequests = mergeRequests
         self.project = project
         self.tags = tags
+        self.build_tags = build_tags
```

Comment:
minor: variable name style mismatch

Reply:
yes, but the style is 50/50 in the code base so it is hard to choose :)

Reply:
kept it as is, since `builder_tags` is written with `_` and this is closely related - when the code style is chaotic then in a very chaotic way this somehow makes sense to me :)

Reply:
@tdctaz @MindWrapper this is the reason why it is what it is.
Twisted prefers !camelCase for variables, functions, and methods (whereas PEP 8 recommends underscores) 
http://twistedmatrix.com/trac/wiki/FrequentlyAskedQuestions#WhydoesntTwistedfollowPEP8 
In general, we follow Twisted camelCase style. But given that the project is in maintenance mode there is no plan for us to fix the code style issues.